container/docker/engine.py

```diff
@@ -780,3 +726,60 @@ def print_version_info(self):
         client = self.get_client()
         pprint.pprint(client.info())
         pprint.pprint(client.version())
+
+    def bootstrap_env(self, temp_dir, behavior, operation, compose_option,
+                      builder_img_id=None, context=None):
```

Comment:
I had done that, but then my IDE threw some warning about using mutable default arguments, and then I arrived at [this](http://docs.python-guide.org/en/latest/writing/gotchas/), so I decided to go with `context=None`.
WDYT?